core/src/main/java/c/org/rajawali3d/control/RenderControl.java

```diff
@@ -0,0 +1,212 @@
+     * 0 if frame are stopped (render thread paused)
+     *
+     * @return {code long} nanoseconds elapsed since most recent start of frame processing
+     */
+    long getFramesElapsedTime();
+
+    //
+    // Scenes population
+    //
+
+    /**
+     * Adds a {@link Scene} to the list of Scene frame delegates. The {@link Scene} will be notified via
+     * {@link Scene#initialize()}, and should take any needed actions.
+     *
+     * @param scene A {@link Scene} to add; duplicates are quietly ignored.
+     */
+    @RequiresRenderTask
+    void addScene(@NonNull Scene scene);
+
+    /**
+     * Removes a {@link Scene} from the list of Scene frame delegates. The {@link Scene} will be notified via
+     * {@link Scene#destroy()}, and should take any needed actions.
+     *
+     * @param scene The {@link Scene} to remove; if not found, it is quietly ignored.
+     */
+    @RequiresRenderTask
+    void removeScene(@NonNull Scene scene);
+
+    //
+    // SceneViews population/ordering
+    //
+
+    /**
+     * Adds a {@link SceneView} to (the end of) the list of SceneView frame delegates. The {@link SceneView} will be
+     * notified via {@link SceneView#onAddToSceneViewControl(RenderControlInternal)}, and should take any needed actions.
+     * <p>
+     * {@link SceneView}s are rendered in list order using a simple back-to-front painter's algorithm (index 0 is
+     * backmost). The added {@link SceneView} will be rendered foremost.
+     * </p>
+     *
+     * @param sceneView The {@link SceneView} to add; duplicates are quietly ignored.
+     */
+    @RequiresRenderTask
+    void addSceneView(@NonNull SceneView sceneView);
+
+    /**
+     * Gets the depth order of this {@link SceneView}'s on-screen viewport relative to that of other {@link SceneView}s.
+     * This is simply the index in the list of SceneView frame delegates, so 0 means back-most, and larger orders are
+     * rendered in front of lower orders.
+     *
+     * @param sceneView
+     * @return
+     */
+    @RequiresRenderTask
+    @IntRange(from = 0)
+    int getSceneViewDepthOrder(@NonNull SceneView sceneView);
+
+    /**
+     * Inserts a {@link SceneView} in the list of SceneView frame delegates at the specified depth order.
+     * The {@link SceneView} will be notified via {@link SceneView#initialize()} ,
+     * and should take any needed actions.
+     * <p>
+     * {@link SceneView}s are rendered in list order using a simple back-to-front painter's algorithm (index 0 is
+     * backmost). The added {@link SceneView} will be rendered foremost.
+     * </p>
+     *
+     * @param sceneView The {@link SceneView} to add; duplicates are quietly ignored.
+     */
+    @RequiresRenderTask
+    void insertSceneView(@NonNull SceneView sceneView, int depthOrder);
+
+    /**
+     * Removes a {@link SceneView} from the list of SceneView frame delegates. The {@link SceneView} will be notified
+     * via {@link SceneView#destroy()}, and should take any needed actions.
```

Comment:
Same contract issue as with `addScene()`

Reply:
And same response...